README.md

```diff
@@ -1,8 +1,8 @@
 Concierge
 =========
 
-[![dockeri.co](http://dockeri.co/image/vshn/concierge)](https://hub.docker.com/r/vshn/concierge/)
 
+[![dockeri.co](http://dockeri.co/image/vshn/concierge)](https://hub.docker.com/r/vshn/concierge/)
```

Comment:
This looks like a mistake stemming from the adaption of the f-strings in the Python script. :hankey:

Reply:
I saw you removed the f's on strings where they weren't needed. Did the change cause some problem?

Reply:
I figured that was in the other repo. I probably move an empty line with the IDE unconsciously. This is now fixed.